### PR TITLE
Adding Kibana to the options

### PR DESCRIPTION
**Pull request by cfontes**
Hi, I added a couple files with a service and a deployment for a kibana with XPACK disabled in order to manage the cluster.

BTW thanks for this, it's a life saver!

### CODE REVIEW

**Comment by cfontes** [comment]
To access it you can use kubernetes dashboard as a proxy, like this

http://localhost:8080/api/v1/proxy/namespaces/default/services/kibana:5601/app/kibana#/discover?_g=()

File: kibana.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+piVersion: extensions/v1beta1
```
Comment (pires) [inline kibana.yaml]: Missing `a`.

Comment (cfontes) [inline kibana.yaml]: Sure, thanks

**Comment by cfontes** [comment]
Hi @pires done.

Could you take a look?

**Comment by pires** [comment]
Thank you, @cfontes!

**Pull request merged**